src/ecalc/libraries/libecalc/common/libecalc/core/models/results/compressor.py

```diff
@@ -127,6 +127,8 @@ class CompressorTrainResult(EnergyFunctionResult):
     stage_results: List[CompressorStageResult]
     failure_status: List[Optional[CompressorTrainCommonShaftFailureStatus]]
     turbine_result: Optional[TurbineResult] = None
+    requested_inlet_pressure: List[Optional[float]]
```

Comment:
Isnt this always set? but as math.nan as a fallback. Can the values ever be `None`?

Reply:
I need to double check this

Reply:
I though so as well. However, when running the TestShowResultsCommand (using show), cached results from an eCalc run are loaded. In this process, empty/null values are converted to None. So, I keep it for now.

Reply:
ah, good point